.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: "Stale label"
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  branches:
+    - dev-es    
+jobs:
+  stale:
+    timeout-minutes: 1
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v7
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'This issue is stale because it has been labeled with inactivity.'
+        stale-pr-message: 'This PR is stale because it has been labeled with inactivity.'
+        exempt-issue-labels: 'maintainers, lang/es'
+        exempt-pr-labels: 'maintainers'
+        stale-pr-label: 'stale'
```

Comment:
- https://github.com/actions/stale#close-issue-label
- https://github.com/actions/stale#close-pr-label

looks useful for maintaining as well.

Reply:
Thank you @seokho-son  for your comments! I considered it, please review it